gprofiler/utils.py

```diff
@@ -266,7 +267,7 @@ def pgrep_exe(match: str) -> List[Process]:
     procs = []
     for process in psutil.process_iter():
         try:
-            if pattern.match(process.exe()):
+            if pattern.match(process_exe(process)):
                 procs.append(process)
         except psutil.NoSuchProcess:  # process might have died meanwhile
```

Comment:
`pgrep_exe()` is unused in the project, so either remove this method, or `except ZombieProcess` as well

Reply:
Keeping it because we might use it later, no reason to remove.

And `ZombieProcess` extends `NoSuchProcess` so it's okay :)

Reply:
Oops, missed it in the description